corehq/apps/data_cleaning/templates/data_cleaning/columns/task_status.html

```diff
@@ -0,0 +1,23 @@
+{% load hq_shared_tags %}
+
+{% if record.percent and record.percent < 100 %}
+  <div
+    class="progress"
+    role="progressbar"
+    aria-label="{% trans_html_attr 'Cleaning progress' %}"
+    aria-valuenow="{{ record.percent }}"
+    aria-valuemin="0"
+    aria-valuemax="100"
+  >
+    <div
+      class="progress-bar progress-bar-striped progress-bar-animated"
+      style="width: {{ record.percent }}%"
+    >
+      {{ record.percent }}%
+    </div>
+  </div>
+{% else %}
+  <span class="badge text-bg-{{ record.status_class }}">
```

Comment:
having the `status_class` be a part of the python makes the full class less grep-able or noticeable by bootstrap migration tools. Is there a way to translate a less css-name specific variable to the full (`text-bg-primary`, etc ) string somewhere in the template itself? I hope that makes sense...

Reply:
yeah, that's a fair point, I can do that in a followup

Reply:
Updated in this PR - 54292f46927